### ISSUE TITLE
Clothes dyeing

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#12464]
Related to https://github.com/space-wizards/space-station-14/issues/12462

The clothesmate should only dispense white clothes (splitting the other stuff out) and there should be an option to have these dyed somewhere (e.g. dedicated machine / uniform printer).

**Comment by SlamBamActionman** [comment]
I don't think we should axe colored clothes from the vending machine, and instead there should be UI improvements made for the vending UI. I do think clothing dying should be supported however.
